### DISCUSSION TITLE
在弹窗里加载viewer关闭弹窗后会报错

### DISCUSSION THREAD

**Discussion by webdq** [zouyaoji/vue-cesium#218] [Q&A] [answered]
包版本：

"cesium": "^1.93.0",
"vue": "^3.2.25",
"vue-cesium": "^3.0.14"

场景：在html中引入cesium的js和css文件，在页面里加载弹窗组件，弹窗里加载viewer

1. 关闭弹窗后会报错

```
Object { name: "DeveloperError", message: "normalized result is not a number", stack: "DeveloperError@http://localhost:3000/static/cesium/Cesium.js:1:452\nCartesian3.normalize@http://localhost:3000/static/cesium/Cesium.js:1:16069 ......
```
[![jZF9qf.png](https://s1.ax1x.com/2022/06/27/jZF9qf.png)](https://imgtu.com/i/jZF9qf)

2. 关闭弹窗并销毁弹窗，外部引入的cesium的js和css会被删除

地址：[GitHub 复现地址](https://github.com/webdq/vite-vue-cesium)

**Comment by zouyaoji** [comment, marked as answer]
在vc-viewer组件上添加:remove-cesium-script="false" 即可避免类似问题。
